module/input/Camera/src/Camera.cpp

```diff
@@ -81,24 +78,35 @@ namespace module::input {
 
                 // Open the camera: Store as shared pointer
                 std::string device_description = arv_get_device_id(device_no);
+                GError* error                  = nullptr;
                 auto camera =
-                    std::shared_ptr<ArvCamera>(arv_camera_new(device_description.c_str()), [](ArvCamera* ptr) {
+                    std::shared_ptr<ArvCamera>(arv_camera_new(device_description.c_str(), &error), [](ArvCamera* ptr) {
                         if (ptr) {
                             g_object_unref(ptr);
                         }
                     });
+                if (error != nullptr) {
+                    g_error_free(error);
+                    error = nullptr;
+                }
```

Comment:
If there was an error shouldn't you be logging it? and maybe taking some corrective action or throwing if there is no corrective action?

Reply:
I was just copying the style used in the other calls to aravis. I can look into that if you want to open an issue.

Reply:
I think you meant to do this?
https://github.com/NUbots/NUbots/blob/cmurtagh/fix_protobuf/module/input/Camera/src/aravis_wrap.hpp#L53-L57

Reply:
I was referring to https://github.com/NUbots/NUbots/blob/main/module/input/Camera/src/settings.hpp#L28, but I will do it how you suggest.